runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/internal/registry/ConfigManager.java

```diff
@@ -224,7 +180,7 @@ else if ("http".equals(configURL.getProtocol()) || "https".equals(configURL.getP
             namespace.id = supplyId.applyAsInt(namespace.name);
 
             // TODO: consider qualified name "namespace::name"
-            namespace.resolveId = name -> name != null ? NamespacedId.id(namespace.id, supplyId.applyAsInt(name)) : 0L;
+            namespace.resolveId = idResolver(namespace);
```

Comment:
I only created this, because the namespace variable has to be effectively final, as we declare it at the beginning of the method now. I just resolved this by making a final copy of namespace.